tests/test_relative_free_energy.py

```diff
@@ -166,13 +170,13 @@ def test_steps_per_frames():
     seed = 2022
     frames = 5
     res = run_vacuum(mol_a, mol_b, core, forcefield, None, frames, seed, n_eq_steps=10, steps_per_frame=2, n_windows=2)
-    assert res.frames[0].shape[0] == frames
+    assert len(res.frames[0]) == frames
```

Comment:
Hmm, to avoid these sorts of changes, I wonder if it make sense to add a `.to_numpy()` method?

(For cases where a `StoredArrays` object can fit entirely in memory, and we would like to interact with it as a single ordinary numpy array)

Reply:
Hmm, we could add this, but we'd have to be careful about where we use it to avoid OOM. (Seems like it would be fine here, but I'm not sure I see a reason to prefer `frames.shape[0]` over `len(frames)`?)

Reply:
(Also, `np.array(frames)` should work)

Reply:
Oh! I overlooked that `np.array(frames)` already worked for this use -- that resolves my comment 👍